astoria/astwifid.py

```diff
@@ -210,7 +210,7 @@ def generate_hostapd_config(self) -> None:
             "bridge": self._bridge,
             "ssid": self._ssid,
             "country_code": self._region,
-            "channel": 7,
+            "channel": 6,
```

Comment:
```suggestion
            "channel": random.choice([1, 6, 11]),
```

We will have a lot of robots at the competition so it's probably a good idea to spread them over several channels.

Ideally we would cache the WiFi channel in `astmetad` so it's persistent between reboots, but I think that's something we can do later

Reply:
We probably want to prioritise the venue wifi over the robot's so having all the robots in one place makes it easier for the venue wifi to adapt onto the other channels.